apps/discoverbos/widget/project/provider.jsx

```diff
@@ -0,0 +1,7 @@
+function Provider({ View, path }) {
+  const data = Social.getr(path, "final");
```

Comment:
And then I extracted the data fetching into a Provider, that just feeds the data into the card.

Reply:
Sure